docusaurus/docs-guides/hsm/yubikey.mdx

```diff
@@ -1,1124 +1,1114 @@
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
-
-# YubiKey by Yubico
-
-[Yubico](https://www.yubico.com/) is a manufacturer of HSM deviceis. A popular line of HSM produced by Yubico is the
-YubiKey. This quickstart guide will use specific device from Yubico - the [YubiKey 5
-nfc](https://www.yubico.com/product/YubiKey-5-nfc).
-
-## Overview
-
-The [YubiKey 5 nfc](https://www.yubico.com/product/YubiKey-5-nfc) is a multi-purpose device with a few different
-security-minded uses. One of the applications on the device is an application called PIV or "Personal Identity
-Verification". PIV is a [standard published by
-NIST](https://csrc.nist.gov/projects/piv/piv-standards-and-supporting-documentation) and describes the kinds of
-credentials which make up the standard. PIV credentials have certificates and key pairs, pin numbers, biometrics
-like fingerprints and pictures, and other unique identifiers. When put together into a PIV credential, it provides
-the capability to implement multi-factor authentication for networks, applications and buildings.
-
-In this quickstart you will see the commands and tools needed to use a [YubiKey 5

```

Comment:
more words changed?

Reply:
The first couple of lines should've been in the other PR. Not pertinent to broken links.

The last line looks like a mistake at a glance because it was probably intended to fix the YubiKey link, not remove the line.

Reply:
i'm confused. i still see a delta here. the only delta i would expect is that top line for the link?

Reply:
The link is the reason I included that hunk, and the hunk includes the delta you are seeing.  That is, Git's hunk selector picked those lines as a patch to accept or reject. I'm sure there's a way to reject individual lines, but I didn't explore it beyond the hunk selectors defaults.